### PR TITLE
Make docker repos configurable via variables

### PR DESCRIPTION
**Pull request by nicklathe**

### CODE REVIEW

File: kubernetes/cloud-configs/controller.yaml [hunk 1]
```diff
@@ -48,7 +49,8 @@ coreos:
           --allow-privileged=true \
           --pod-manifest-path=/etc/kubernetes/manifests \
           --cluster_dns=${kubernetes_dns_service_ip} \
-          --cluster_domain=cluster.local
+          --cluster_domain=cluster.local \
+          --pod-infra-container-image ${pod_infra_image_repo}:${pod_infra_version}
```
Comment (weitenghuang) [inline kubernetes/cloud-configs/controller.yaml]: Can we keep same format with `=` between flag and value?

File: kubernetes/cloud-configs/node.yaml [hunk 2]
```diff
@@ -32,7 +33,8 @@ coreos:
           --cluster_domain=cluster.local \
           --kubeconfig=/etc/kubernetes/node-kubeconfig.yaml \
           --tls-cert-file=/etc/kubernetes/tls/node.pem \
-          --tls-private-key-file=/etc/kubernetes/tls/node-key.pem
+          --tls-private-key-file=/etc/kubernetes/tls/node-key.pem \
+          --pod-infra-container-image ${pod_infra_image_repo}:${pod_infra_version}
```
Comment (weitenghuang) [inline kubernetes/cloud-configs/node.yaml]: Use `=` ?

File: kubernetes/cloud-configs/controller.yaml (hunk 1 above)
Comment (nicklathe) [reply kubernetes/cloud-configs/controller.yaml]: Will update PR.

File: kubernetes/cloud-configs/node.yaml (hunk 2 above)
Comment (nicklathe) [reply kubernetes/cloud-configs/node.yaml]: Will update.

**Pull request merged**